frontend/src/app/constants.js

```diff
@@ -16,3 +16,9 @@ export const BTN_STYLES = {
   PRIMARY: "Main",
   SECONDARY: "Secondary",
 };
+
+export const ICON_STYLES = {
+  THEME: "theme",
+  LANGUAGE: "language",
+  INFO: "info",
+};
```

Comment:
Загалом в константи можна заностит одразу той текст який будеш виводити користувачу, потім його легше знайти і віредагувати..

До прикладу 
```suggestion
export const SETTINGS_ITEM_STYLES = {
  THEME: "Тема",
  LANGUAGE: "Мова інтерфейсу",
  INFO: "Правила спільноти",
};
```
І цю константу всервіно можна використовувати далі в умовах...
Але й у неймінгу не може бути слово іконка, тому якщо для іконок треба теж костанту то розділиш